index.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css"
+    />
+    <title>Portfolio</title>
+</head>
+<body>
+    <section>
+        <nav id="navbar" class="navbar fixed-top navbar-expand-lg navbar-dark bg-secondary">
+          <div class="container-fluid">
+            <a class="navbar-brand" href="#">John Doe</a>
+            <button class="navbar-toggler" 
+              type="button" 
+              data-bs-toggle="collapse" 
+              data-bs-target="#navbarNavAltMarkup"
+              aria-controls="navbarNavAltMarkup"
+              aria-expanded="false"
+              aria-label="Toggle navigation"
+            >
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+              <div class="navbar-nav">
+                <a class="nav-link text-white" href="#portfolio">Portfolio</a>
+                <a class="nav-link text-white" href="#about">About</a>
+                <a class="nav-link text-white" href="#contact">Contact</a>
+                <i class="bi bi-envelope text-white"></i>
+              </div>
+            </div>
+          </div>
+        </nav>
+    </section>
+      <!-- HOME SECTION -->
+      <section class="container-fluid bg-dark pt-5 pt-lg-0">
+        <div class="row py-5 py-lg-2">
+          <div class="col-lg-1 d-flex align-items-center order-3 order-lg-0">
+            <ul class="tags flex-lg-column flex-md-row">
+              <li class="elements">
+                <a href="#">
+                  <i class="bi bi-twitter text-white"></i>
+                </a>
+              </li>
+              <li class="mb-2 me-2">
+                <a href="#">
+                  <i class="bi bi-linkedin text-white"></i>
+                </a>
+              </li>
+              <li class="mb-2 me-2">
+                <a href="#">
+                  <i class="bi bi-instagram text-white"></i>
+                </a>
+              </li>
+              <li class="mb-2 me-2">
+                <a href="#">
+                  <i class="bi bi-github text-white"></i>
+                </a>
+              </li>
```

Comment:
- [ ] Kindly style icons to have the same spacing and alignment like the Figma design, see screenshot above 👍🏿